app/helpers/application_helper.rb

```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title = ""
```

Comment:
```suggestion
  def full_title page_title
```
ri dc rồi, ko cần set giá trị mặc định cho parram page_title đâu bé